README.md

```diff
@@ -64,16 +64,48 @@ jekyll serve --watch --config _config.yml,_config-dev.yml
 
 ### Updating
 
-Updating the various repos can be done with running the this script
+Updating the various repositories can be done with running the this script
 
 ```
 ./scripts/update.sh
 ```
 
 [osd-net]: http://opensourcedesign.net
-[osd-prose]: http://edit.opensourcedesign.net
 [osd-org]: https://github.com/opensourcedesign/
 [jekyll]: https://jekyllrb.com
 [bootstrap]: https://getbootstrap.com
 [installing-ruby]: https://www.ruby-lang.org/en/documentation/installation/
 [github-ssh]: https://help.github.com/articles/connecting-to-github-with-ssh/
+
+
+## 👩‍🚀 Contributors, backers & sponsors
+
+This project exists thanks to all the **people who contribute**.
+<a href="graphs/contributors"><img src="https://opencollective.com/opensourcedesign/contributors.svg?width=890&button=false" /></a>
+
+Thank you to **all our backers**! 🙏 ([Become a backer](https://opencollective.com/opensourcedesign#backer))
+
+<a href="https://opencollective.com/opensourcedesign#backers" target="_blank"><img src="https://opencollective.com/opensourcedesign/backers.svg?width=890"></a>
+
+**Support this project by becoming a sponsor.** Your logo will show up here with a link to your website. ([Become a sponsor](https://opencollective.com/opensourcedesign#sponsor))
+
+<a href="https://opencollective.com/opensourcedesign/sponsor/0/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/0/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/1/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/1/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/2/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/2/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/3/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/3/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/4/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/4/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/5/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/5/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/6/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/6/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/7/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/7/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/8/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/8/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/9/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/9/avatar.svg"></a>
```

Comment:
The sponsor has a weird spacing in my preview

Reply:
Don’t think this can be changed as it comes from Open Collective defaults. :\